facilitator/src/bin/facilitator.rs

```diff
@@ -1031,7 +1031,8 @@ fn batch_signing_key_from_arg(matches: &ArgMatches) -> Result<BatchSigningKey> {
         .value_of("batch-signing-private-key-identifier")
         .unwrap();
     Ok(BatchSigningKey {
-        key: EcdsaKeyPair::from_pkcs8(&ECDSA_P256_SHA256_ASN1_SIGNING, &key_bytes)?,
+        key: EcdsaKeyPair::from_pkcs8(&ECDSA_P256_SHA256_ASN1_SIGNING, &key_bytes)
```

Comment:
I can probably do this in another PR if we don't want to take it here

Reply:
No, this a much better error message, and is useful to have here. However I would just do `.context("failed to parse batch signing key")?`. There's no need for a closure since you're not capturing any variables here.